src/workerd.rs

```diff
@@ -59,17 +53,12 @@ pub enum ServerlessError {
 
 async fn get_current_deposit(
     tx_hash: &str,
-    rpc: &str,
-    billing_contract_add: &str,
+    billing_contract: &BillingContract<Provider<Http>>,
```

Comment:
`BillingContract<Provider<Http>>` is used multiple times. Better to define this type, use that new type everywhere. It would be concise.